x-pack/legacy/plugins/siem/public/containers/detection_engine/rules/use_rule_status.test.tsx

```diff
@@ -72,104 +72,108 @@ describe('useRuleStatus', () => {
     cleanup();
   });
 
-  test('init', async () => {
-    await act(async () => {
-      const { result, waitForNextUpdate } = renderHook<string, ReturnRuleStatus>(() =>
-        useRuleStatus('myOwnRuleID')
-      );
-      await waitForNextUpdate();
-      expect(result.current).toEqual([true, null, null]);
-    });
-  });
-
-  test('fetch rule status', async () => {
-    await act(async () => {
-      const { result, waitForNextUpdate } = renderHook<string, ReturnRuleStatus>(() =>
-        useRuleStatus('myOwnRuleID')
-      );
-      await waitForNextUpdate();
-      await waitForNextUpdate();
-      expect(result.current).toEqual([
-        false,
-        {
-          current_status: {
-            alert_id: 'alertId',
-            last_failure_at: null,
-            last_failure_message: null,
-            last_success_at: 'mm/dd/yyyyTHH:MM:sssz',
-            last_success_message: 'it is a success',
-            status: 'succeeded',
-            status_date: 'mm/dd/yyyyTHH:MM:sssz',
-            gap: null,
-            bulk_create_time_durations: ['2235.01'],
-            search_after_time_durations: ['616.97'],
-            last_look_back_date: '2020-03-19T00:32:07.996Z',
-          },
-          failures: [],
-        },
-        result.current[2],
-      ]);
-    });
-  });
-
-  test('re-fetch rule status', async () => {
-    const spyOngetRuleStatusById = jest.spyOn(api, 'getRuleStatusById');
-    await act(async () => {
-      const { result, waitForNextUpdate } = renderHook<string, ReturnRuleStatus>(() =>
-        useRuleStatus('myOwnRuleID')
-      );
-      await waitForNextUpdate();
-      await waitForNextUpdate();
-      if (result.current[2]) {
-        result.current[2]('myOwnRuleID');
-      }
-      await waitForNextUpdate();
-      expect(spyOngetRuleStatusById).toHaveBeenCalledTimes(2);
-    });
-  });
-
-  test('init rules statuses', async () => {
-    const payload = [testRule];
-    await act(async () => {
-      const { result, waitForNextUpdate } = renderHook<string, ReturnRulesStatuses>(() =>
-        useRulesStatuses(payload)
-      );
-      await waitForNextUpdate();
-      expect(result.current).toEqual({ loading: false, rulesStatuses: [] });
+  describe('useRuleStatus', () => {
```

Comment:
Note: Didn't make any changes to the tests themselves, just separated them out into their own describe blocks for readability purposes.